extractor/cli/go-autobuilder/go-autobuilder.go

```diff
@@ -206,6 +207,51 @@ func checkVendor() bool {
 	return true
 }
 
+func getOsToolsSubdir() (string, error) {
+	switch runtime.GOOS {
+	case "darwin":
+		return "osx64", nil
+	case "linux":
+		return "linux64", nil
+	case "windows":
+		return "win64", nil
+	}
+	return "", errors.New("Unknown OS: " + runtime.GOOS)
```

Comment:
Personally I like the `fmt.Errorf` more than `errors.New`, but that's entirely a style thing. Also, I think "unsupported" is more accurate here.

```suggestion
	return "", fmt.Errorf("Unsupported OS: %s", runtime.GOOS)
```